packages/connect-core/src/transactions/TransactionIntent.ts

```diff
@@ -1,78 +0,0 @@
-import { providers as ethersProviders } from 'ethers'
```

Comment:
I finally remove the `Intent` as a class because was an intermediary step that didn't made a lot of sense and just produce a more convoluted API.